src/docfx/build/Build.cs

```diff
@@ -143,6 +144,22 @@ private static async Task BuildFile(Context context, Document file, TableOfConte
                 Console.WriteLine($"Build {file.FilePath} failed");
                 throw;
             }
+
+            bool ShouldBuildFile()
+            {
+                if (file.ContentType == ContentType.TableOfContents)
```

Comment:
Does a repo always have a Toc file? If yes, `TocMap` will always be loaded, seems no need to lazy load?